scripts/travis/build-all-in-one-image.sh

```diff
@@ -2,15 +2,24 @@
 
 set -e
 
-BRANCH=${BRANCH:?'missing BRANCH env var'}
-
-source ~/.nvm/nvm.sh
-nvm use 10
-make build-ui
+if [[ -z "$BRANCH" ]]; then
+  BRANCH=$(git rev-parse --abbrev-ref HEAD)
+  echo "BRANCH env var not defined, using current branch $BRANCH instead ..."
+fi
```

Comment:
what is prompting this change?

Reply:
Instead of falling back to `missing BRANCH env var`, it's way better to fall back to the actual local branch name.